### DISCUSSION TITLE
Adding extra packages (slices) to Ubuntu chiseled images

### DISCUSSION THREAD

**Discussion by Daniel15** [dotnet/dotnet-docker#5012] [Q&A] [answered]
I'd like to use the Ubuntu chiseled `runtime-deps` or `aot` images, but need to add a few more packages. Is forking the Docker file (https://github.com/dotnet/dotnet-docker/blob/8654408b4f72027d5411650110bb9d83c1d50ce0/src/runtime-deps/8.0/jammy-chiseled/amd64/Dockerfile) the best approach, or is there something I can do to layer a few extra slices on top of the existing base images?

Thanks!

**Comment by akoken** [comment]
Yes, it's the best and only option you have right now. 
#4527

**Comment by lbussell** [comment, marked as answer]
Hi @Daniel15, we are currently working out what this documentation looks like. Ubuntu has created a recommended pattern [here](https://github.com/ubuntu-rocks/dotnet/#building-the-net-application-image). We haven't recommended that pattern yet because it doesn't update the dpkg cache file, meaning that vulnerability scanning tools will give inaccurate results for your image.

However, a workaround to merge dpkg status files between layers is documented in [this comment](https://github.com/ubuntu-rocks/dotnet/issues/21#issuecomment-1815937307) in the Ubuntu repo. We haven't tested it out yet, but if the dpkg status merging maintains the correct image scanning functionality I would recommend that approach. I'll be looking at this soon and working on some official documentation if necessary. You can track this in https://github.com/dotnet/dotnet-docker/issues/4811.

**Reply by Daniel15** [reply]
Thank you! I'm excited to try native AoT plus the chiseled images. Thanks for working on this! 😃
